### ISSUE TITLE
[BUG] A lot of "route.continue" in allure report after successful test run

### ISSUE THREAD

**Issue by qakos** [microsoft/playwright#10098]
**Context:**
- Playwright Version: [1.16.2, 1.16.3]
- Operating System: [Windows 10, Linux]
- Node.js version: [16.13]
- Browser: [All, Chromium, Firefox, WebKit]
- Extra: [any specific details about your environment]

**Code Snippet**

Help us help you! Put down a short code snippet that illustrates your bug and
that we can run and debug locally. For example:

```javascript
// @ts-check
const { test, expect } = require('@playwright/test');

const {
  DEV_LOGIN_PAGE_URL,
  DEV_MAIN_PAGE_URL,
} = require('../framework/constants');

const { LoginPage, MainPage } = require('../framework');

test.beforeEach(async ({ page }) => {
  // interception of yandex metric requests
  page.route('**', route => {
    const request = route.request();
    // console.log('request url: ', request.url());
    if (request.url().match('yandex') || request.url().match('metrika')) {
      // eslint-disable-next-line no-console
      console.log('blocked by client');
      return route.abort('blockedbyclient');
    }

    return route.continue();
  });

  await page.goto(DEV_LOGIN_PAGE_URL);
});

test('Case', async ({ page }) => {
  const loginPage = new LoginPage(page);
  const mainPage = new MainPage(page);

  await loginPage.loginAsUser();
  await mainPage.clickConfirmContractButton();
  await expect(mainPage.page.url()).toEqual(DEV_MAIN_PAGE_URL);
});
```

**Describe the bug**

Add any other details about the problem here.
A lot of "route.continue" in allure report after successful test run
![photo_2021-11-05_22-06-20](https://user-images.githubusercontent.com/93794669/140572194-0d52f149-492f-4790-a40b-2a27ece4864c.jpg)

**Comment by mxschmitt** [comment]
@pavelfeldman this should be fixed in ToT already?

**Comment by pavelfeldman** [comment]
This is now fixed in the HTML report, but it requires a separate fix in Allure. Please file this bug in the allurejs repo and someone can fix it there.

**Comment by alex-vungle** [comment]
Hi @pavelfeldman , sorry for disturbing you in a closed ticket. May I know how did you fix this issue? 

Currently, I'm still seeing the **route.continue()** in the HTML report, the only difference compared with the allure report is in the HTML report, those **route.continue()** are grouped together.

**Comment by pavelfeldman** [comment]
Please file a new issue if this has regressed!
